src/dp_util.c

```diff
@@ -89,7 +89,7 @@ static int get_num_of_vfs_pattern(void)
 
 int dp_get_num_of_vfs(void)
 {
-	int vfs = dp_conf_get_nic_type() == DP_CONF_NIC_TYPE_TAP
```

Comment:
And here it should then mean "sriov is only supported for 'hw' option"

Reply:
Yeah, I can turn the conditions around in both cases (also the one above), it would look more readable and nicer.